tasks/command/index.ts

```diff
@@ -13,7 +29,7 @@ async function main(): Promise<void> {
             return x.length;
         });
 
-    var result = await task.exec(FLUTTER_TOOL_PATH_ENV_VAR, splittedArgs);
+    var result = await task.exec(flutterPath, splittedArgs);
```

Comment:
change it back to FLUTTER_TOOL_PATH_ENV_VAR

Reply:
That won't work if I also make the two changes you requested above.  FLUTTER_TOOL_PATH_ENV_VAR ends up being the constant string 'FlutterToolPath' which doesn't resolve to an executable.

If you are wanting to specify the binary as the first argument in the `arguments` input then I think it either should be a separate `command` input and `arguments` left as is, or arguments should be renamed to `script`.  And if the latter, I would question what value FlutterCommand then has over the generic CmdLine task and example given in the FAQs.

Reply:
@md84419  I see what you mean. Leave this code as is and add the `flutterDirectory` input to task.json

```dart

        {
            "name": "flutterDirectory",
            "type": "filePath",
            "label": "Path to the Flutter SDK",
            "defaultValue": "",
            "helpMarkDown": "Path to the Flutter SDK if not using Flutter Install task before this one.",
            "required": false
        },
```

So now it will work without adding `$fluttertoolpath` just like build task. So now it will be just `pub get` or `pub outdated --prereleases`